app/(app)/(purchase-and-payment)/ticket-purchase.tsx

```diff
@@ -77,16 +86,20 @@ const TicketPurchasePage = () => {
       {isPending || data?.paymentStatus === 'PENDING' ? (
         <ContentWithAvatar title="Ticket is being processed" text={ticketId} />
       ) : isError || data.paymentStatus === 'FAIL' ? (
-        <ContentWithAvatar variant="error" title={t('paymentFailed')} text={t('paymentFailedText')}>
+        <ContentWithAvatar
+          variant="error"
+          title={t(getTranslationName('Failed'))}
+          text={t(getTranslationName('FailedText'))}
```

Comment:
Or maybe better would be keeping the `text` as nested key:
```js
const translationKey = isProlongation ? 'prolongation' : 'payment'

t(`${translationKey}.failed`)
t(`${translationKey}.failedText`)
```
so that it's easily visible what translation keys are used.